src/P5Functions/Math.js

```diff
@@ -433,7 +433,14 @@ module.exports = function withMath(p, undef) {
       return internalRandomGenerator() * arguments[0];
     }
     var aMin = arguments[0], aMax = arguments[1];
-    return internalRandomGenerator() * (aMax - aMin) + aMin;
+    while (true) {
```

Comment:
I have no problem with equating 100 to infinity.  However, if execution falls out of the loop, the function needs to return a legitimate value (like `aMin`).

The bug report explains how and why the computation yields the wrong result.  If we could tell the FU to round an addition towards zero, then then there would be no reason to retry.  My understanding is that we cannot get to the FU API in this environment.